### PR TITLE
[IMP] mail: new tooltip

### PR DESCRIPTION
**Pull request by igbe-odoo**
Add a new tooltip on the popout to open a pdf in a different window.

task-4423155

### CODE REVIEW

**Review by RicGR98 — APPROVED** [review]
@robodoo r+

File: addons/mail/static/src/core/common/attachment_view.xml [hunk 1]
```diff
@@ -3,7 +3,12 @@
 
     <t t-name="mail.AttachmentView">
         <div t-if="state.thread.attachmentsInWebClientView.length > 0" class="o-mail-Attachment">
-            <div class="o_attachment_control popout d-print-none" t-on-click="onClickPopout"><i class="fa fa-window-restore" aria-hidden="Pop out" title="Pop out"/></div>
+            <div class="o_attachment_control popout d-print-none cursor-pointer"
+                 t-on-click="onClickPopout"
+                 data-tooltip="Pop the preview in a separate window for dual screens."
```
Comment (dylankiss) [inline addons/mail/static/src/core/common/attachment_view.xml:8]: I think the wording is a bit weird. Maybe something simpler like this?
```suggestion
                 data-tooltip="Open preview in a separate window."
```

Comment (dylankiss) [reply addons/mail/static/src/core/common/attachment_view.xml:8]: "Pop the preview" and "for dual screens" is just incorrect English 🤷

Comment (william-andre) [reply addons/mail/static/src/core/common/attachment_view.xml:8]: Odooglish at it again :laughing:

**Comment by RicGR98** [comment]
@robodoo r-

While waiting to decide if it's good or bad English (not my strong suit, perhaps best to listen to dyki since he is in the translation team)

@robodoo delegate+

If you decide to change it (or not)

File: addons/mail/static/src/core/common/attachment_view.xml (hunk 1 above)
Comment (igbe-odoo) [reply addons/mail/static/src/core/common/attachment_view.xml:8]: Oopsie doopsie, well, I'll change that then, thanks you :smile:

**Comment by igbe-odoo** [comment]
@robodoo r+

**Pull request closed without merging**